.infra/rdev/values.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+  explorer:
+    image:
+      tag: sha-652d1618
```

Comment:
this is the current container tag used for deployement. Whatever is checked into the repo is what is deployed